client/src/app/shared/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,26 @@
+<nav class="sidebar bg-light" *ngIf="isAuthenticated" [ngClass]="{'sidebar-collapsed': isCollapsed}">
+    <ul class="navbar-nav">
+      <li class="nav-item" *ngIf="isAuthenticated">
```

Comment:
```suggestion
      <li class="nav-item">
```

Not needed, also not needed below checks for `*ngIf="isAuthenticated"`

Reply:
Fixed